python/smt_switch/enums_imp.pxi

```diff
@@ -151,10 +151,6 @@ solverattr = ModuleType('solverattr')
 _add_module(solverattr)
 solverattr.__file__ = FILENAME
 
-cdef SolverAttribute LOGGING = SolverAttribute()
-LOGGING.sa = c_LOGGING
-setattr(solverattr, "LOGGING", LOGGING)
-
 cdef SolverAttribute TERMITER = SolverAttribute()
 TERMITER.sa = c_TERMITER
 setattr(solverattr, "TERMITER", TERMITER)
```

Comment:
Assuming this new `get_op` attribute is indeed added to c++, shouldn't it be added to python?

Reply:
Yep, but I decided to remove it based on your feedback so we're good!